src/lib.rs

```diff
@@ -20,13 +20,23 @@
 //!    format!("{:30}", "format works as expected. This will be padded".blue());
 //!    format!("{:.3}", "and this will be green but truncated to 3 chars".green());
 //!
+//! Custom colours are implemented using the `rgb` crate, which is re-exported for
```

Comment:
Are we using UK English or American English in the rest of our docs? :laughing:

Reply:
well if the UK English is an option....